doc/PRIVACY-POLICY.md

```diff
@@ -0,0 +1,33 @@
+# Privacy policy documentation
+
```

Comment:
This document describes various privacy consideration that should be considered when deploying an instance of demarches-simplifiees.fr.